datadog/dogstatsd/base.py

```diff
@@ -418,16 +428,17 @@ def _is_telemetry_flush_time(self):
         return False
 
     def _send_to_server(self, packet):
-        flush_telemetry = self._is_telemetry_flush_time()
-        if flush_telemetry:
-            packet += "\n"+self._flush_telemetry()
+        self._xmit_packet(packet)
+        if self._is_telemetry_flush_time():
+            telemetry = self._flush_telemetry()
+            self._reset_telemetry()
+            self._xmit_packet(telemetry)
 
+    def _xmit_packet(self, packet):
```

Comment:
Why `_xmit_packet` ? Is this a typo for `_emit_packet` or does it means something ?

Reply:
Most of the things I worked before used the wording `xmit` as a short version of transmit when sending a packet, [linux kernel does too](https://elixir.bootlin.com/linux/latest/source/include/linux/netdevice.h#L97), I have no strong opinion about `emit` vs. `xmit` vs. `transmit` in this context.